server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+
+<div th:fragment="cardsSection (cards)">
+  <ul class="usa-card-group">
+    <th:block th:each="card : ${cards}">
+      <li
+        th:replace="~{applicant/ProgramCardsSectionFragment :: card(${card})}"
+      ></li>
+    </th:block>
+  </ul>
+</div>
+
+<li
+  th:fragment="card (card)"
+  class="usa-card tablet-lg:grid-col-6 widescreen:grid-col-4 cf-application-card"
+>
+  <div class="usa-card__container">
+    <div class="usa-card__header">
+      <h2 class="usa-card__heading" th:text="${card.getTitle()}"></h2>
+    </div>
+    <div class="usa-card__body">
+      <p th:text="${card.getBody()}">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis earum
```

Comment:
this should be removed. It conflicts with the th:text value, so it isn't used

Reply:
Done